README.md

```diff
@@ -252,9 +366,128 @@ Parks and Recreation has 7 seasons.
 An ice cream cone costs $00000000002.99
 ```
 
+NOTE: Without the `%n` format specifier, the output would all be on one line!
+
+## Code Along
+
+Let's transform a small example to line up 3 lines of output in two columns.
+Create a new class in IntelliJ (or use the browser-based Java Visualizer) and copy the following code:
+
+```java
+public class StringPrecisionExample {
+    public static void main(String[] args) {
+
+        // The first string in each format string should be left aligned with a fixed width of 10 characters.
+        // The prices should be formatted with 2 digits after the decimal point.
+
+        String columnHeading = "%s %s";
+        String productFormat = "%s $%f";
+
+        System.out.printf(columnHeading, "PRODUCT", "PRICE");
+        System.out.printf(productFormat, "Potatoes", 4.27);
+        System.out.printf(productFormat, "Eggs", 3.99 );
+    }
+}
+```
+
+We would like the program to output:
+
+```text
+PRODUCT    PRICE
+Potatoes   $4.27
+Eggs       $3.99
+```
+
+However, it currently prints:
+
+```text
+PRODUCT PRICEPotatoes $4.270000Eggs $3.990000
+```
+
+### TASK #1
+
+<details>
+    <summary>
+    Update the <code>columnHeading</code> and <code>productFormat</code>
+    format strings to add a newline specifier at the end of the string.
+    </summary>
+
+  <p>ANSWER:<br><code>String columnHeading = "%s %s%n";</code><br><code>String productFormat = "%s $%f%n";</code></p> 
+
+</details>
+
+Confirm the program prints three separate lines of output:
+
+```text
+PRODUCT PRICE
+Potatoes $4.270000
+Eggs $3.990000
+```
+
+### TASK #2
+
+<details>
+    <summary>
+    Update the <code>productFormat</code>
+    format string to format the price with display 2 digits after the decimal point.
```

Comment:
"Update the `productFormat` format string to format the price with display 2 digits after the decimal point."

Either remove the "display" or rewrite to: "Update the `productFormat` format string to display only 2 digits after the decimal point."

Reply:
fixed